### ISSUE TITLE
Uninstall on macOS

### ISSUE THREAD

**Issue by mkyrilov** [containers/podman-desktop#1507]
I wanted to evaluate Podman. I am using a MacBook Pro M2. I installed Podman Desktop as per the instructions [here](https://podman-desktop.io/docs/Installation/macos-install#installing-podman-desktop-on-macos-using-dmg-file). I picked the `.dmg` option, not Homebrew. I now am done evaluating Podman and I wish to uninstall it from my system. The only instructions I am able to find are for Homebrew installations.

Does anyone know how to uninstall Podman, Podman Desktop and all the login and background items it has installed?

I found [this](#11319) issue which is similar, but again only solutions for Homebrew installations are provided.

**Comment by benoitf** [comment]
for podman binaries:
 `sudo rm -rf /opt/podman`

for podman-desktop binaries
`rm -rf "/Applications/Podman Desktop"`
`rm ~/Library/LaunchAgents/io.podman_desktop.PodmanDesktop.plist`

for podman & podman-desktop config
`rm -rf ~/.config/containers`
`rm -rf ~/.local/share/containers`

**Comment by mkyrilov** [comment]
@benoitf Thank you for the fast response.

The instructions you provided removed almost everything. Only 1 item left. A background task from Red Hat, Inc. This was added when I first launched the Podman Desktop app. Any suggestions on how to remove that?

<img width="827" alt="Screenshot 2023-02-16 at 1 21 49 PM" src="https://user-images.githubusercontent.com/2027390/219490355-abfd168e-4d8e-446c-b557-f0728aaea8d6.png">
